src/main/java/io/reactiverse/pgclient/impl/codec/DataTypeCodec.java

```diff
@@ -1287,6 +1293,48 @@ private static byte decodeHexChar(byte ch) {
     return (byte)(((ch & 0x1F) + ((ch >> 6) * 0x19) - 0x10) & 0x0F);
   }
 
+  private static boolean isHexFormat(int index, int len, ByteBuf buff) {
```

Comment:
do we have test with empty bytea string

Reply:
added in https://github.com/reactiverse/reactive-pg-client/pull/241/commits/4b002d82a99295723a2427ea289e78a2558005ed